frontend/src/app/map-view/comparison/comparison.component.html

```diff
@@ -5,34 +5,70 @@ <h2>{{ 'map.compare.title' | translate }}</h2>
     <form [formGroup]="compareForm" (ngSubmit)="submit()">
       <article>
         <h3>{{'map.compare.base-scenario' | translate }}</h3>
-        <span class="suggestion">{{'map.compare.base-scenario-suggestion' | translate }}</span>
-        <mat-select #base [placeholder]="'map.compare.choose-scenario' | translate" formControlName="a"
-                    (valueChange)="changeBase($event)" required="true" [panelClass]="{ 'ddPanel cmp': true }">
-          <mat-option *ngFor="let c of calculations$ | async" [value]="c.id"
-                      [ngClass]="{ 'purged' : c.isPurged }">{{ c.name }}</mat-option>
-        </mat-select>
+        <mat-radio-group class="set-implicit-wrapper" (change)="setImplicit($event)">
+          <mat-radio-button class="small"
+                            [value]="true"
+                            [checked]="useImplicit">{{
+            'map.compare.implicit-baseline' | translate
+          }}</mat-radio-button>
+          <mat-radio-button class="small"
+                            [value]="false"
+                            [checked]="!useImplicit">{{
+            'map.compare.explicit-baseline' | translate
+          }}</mat-radio-button>
+        </mat-radio-group>
+        <div class="select-base-wrapper" [ngClass]="{'hidden' : useImplicit }">
```

Comment:
Yes. I think this way is preferable because the wrapper is hidden by the click of a radio button. The inner drop down control contains a list which would need to be repopulated from state  by a rxjs selector. If anything, to me it simply feels a bit more 'economical' to hide it with styling.